CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 ### Fixed
 
 - UI: Fixed Live Document Capture flow's camera inactive warning not displaying the basic camera fallback option if `uploadFallback` is not defined for SDK configuration's Document step.
+- UI: Fixed Stick Hover State for buttons on iOS Safari
```

Comment:
This needs to be moved under the `## [next-version]` header now as we have just released the other changes